examples/vpc_lattice/main.tf

```diff
@@ -0,0 +1,78 @@
+
+
+# VPC module
+module "vpc" {
+  #source  = "aws-ia/vpc/aws"
+  #version = ">= 4.3.0"
+  source = "../.."
```

Comment:
Please flip these so that the relative source is commented and the Terraform Registry source & version constraint are uncommented so that customers can copy & paste from the TF Registry. Afterward, please rebuild the terraform-docs.